api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -207,6 +197,23 @@ private void ensureConceptMapTypeIsSet(Concept concept) {
 		}
 	}
 
+	private void voidNameMakeSynonym(ConceptName conceptName) {
```

Comment:
The method name seems to suggest that you are also voiding the name. Can you change it to match the comments?